tensorflow/core/kernels/nn_ops_test.cc

```diff
@@ -112,6 +112,9 @@ static void BM_ConvFloat(::testing::benchmark::State& state, int batch,
   if (!IsGoogleCudaEnabled() && use_gpu) {
     state.SetLabel(
```

Comment:
This label now no longer has any affect.  Can we remove these?

Reply:
@cantonios Thanks for the suggestion. I have removed those redundant statements.